packages/components/menu/src/menu-button.tsx

```diff
@@ -23,7 +23,7 @@ export const MenuButton = forwardRef<MenuButtonProps, "button">(
     const { onKeyDownRef, onUpstreamRestoreFocus } = useUpstreamMenuItem() ?? {}
     const {
       buttonRef,
-      isOpen,
+      isOpen: open,
```

Comment:
The `useMenu` is a hook used internally. So, `isOpen: open` is unnecessary. Please use `open`.

Reply:
Fixed in the commit [`06e6781` (#3693)](https://github.com/yamada-ui/yamada-ui/pull/3693/commits/06e6781b87f82567d837cec6ee04ab4a8714a944) !